.mergify.yml

```diff
@@ -92,3 +92,11 @@ pull_request_rules:
       - files~=^testing/environments/snapshot.*\.yml$
     actions:
       delete_head_branch:
+  - name: delete upstream branch after merging changes on .go-version
+    conditions:
+      - merged
```

Comment:
What happens when I close the PR? Is the branch deleted?

Reply:
No, there is actually a `conditions: -closed` that I was not aware of :)